tests/test_immutabledict.py

```diff
@@ -160,3 +162,14 @@ def test_immutabledict_duplication_blocking(self):
         immutabledict_from_unique_keys(good)
         immutabledict((x for x in good), forbid_duplicate_keys=True)
         immutabledict_from_unique_keys(x for x in good)
+
+    def test_inverse(self):
+        output = immutabledict({"foo": "bar", "bar": "bat"})
+        self.assertEqual(output.inverse(), immutabledict({"bar": "foo", "bat": "bar"}))
```

Comment:
Rather than assign to `output`, put the dict definition directly in the `assertEqual` call.